apps/showcase/src/assets/trainings/sdk/shared/training-sdk-app.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../../../../../schemas/webcontainer-file-system-tree.schema.json",
+  "fileSystemTree": {
```

Comment:
This is rather specific, this will not be shared with all the exercises. It should probably be generated from the exercise folder.

Reply:
Should we have it versioned then in a similar fashion as the sdk? It will be easier to update the content and the dependencies

Reply:
it would be better to be versioned as files instead of JSON yes, as for `monorepo-template.json`
this can be done in a separate PR for both

Reply:
moved to exercise